imports/plugins/core/product/server/no-meteor/schemas/schema.graphql

```diff
@@ -6,3 +6,68 @@ type ProductConfiguration {
   "The ProductVariant ID"
   productVariantId: ID!
 }
+
+"Operation"
+type Operation {
+  "Id of document"
+  _id: String
+}
+
+"A bulk write error type"
+type WriteError {
+  "Error code"
+  code: Int
+
+  "Error message"
+  errmsg: String
+
+  "Index of of error"
+  index: Int
```

Comment:
Is this useful information? Not sure what it's for. Maybe could be removed. If not, the description has "of of" so need to fix that

Reply:
clean up the error messages to only include the `documentId` and `errorMsg` for write errors.